### DISCUSSION TITLE
Shelly Addon and Shelly plus 1 compatible with Tasmota?

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#22255] [Support request] [answered]
Hello, I want to order the Shelly addon with a temperature sensor. Does it work with a Shelly plus1 Tasmota 14.2.0?

**Comment by sfromis** [comment]
Well, Tasmota still works if you plug it in. What you can use it for depends on what you'd want to plug into the addon, like what "temperature sensor" means.

This sample template does indicate a setup for one type of temperature sensor.
https://templates.blakadder.com/shelly_plus_add-on.html

**Comment by Noschvie** [comment, marked as answer]
![grafik](https://github.com/user-attachments/assets/a28dc09a-42f8-434d-9931-77135e212b58)
A screenshot says more than many words

**Reply by dippes** [reply]
Exactly the D18B20 temperature sensor.

Do you have to laboriously calibrate it?

**Reply by Noschvie** [reply]
No, it wasn't necessary.
